users/tests.py

```diff
@@ -1385,7 +1386,7 @@ def test_get_assessee_assessor_with_google_matching_data_when_no_assessee_and_as
 
     @patch.object(google_login, 'create_assessee_from_data_using_google_auth')
     def test_register_assessee_with_google_data_when_not_exist(self, mocked_create_assessee_google):
-        google_login.register_assessee_with_google_data(self.dummy_user_data)
+        google_login.login_or_register_assessee_with_google_data(self.dummy_user_data)
```

Comment:
Hello, Jo. Based on this, is the login and registration conducted on the same endpoint?

Reply:
Noted, Jo. Thanks for the udate.